package.json

```diff
@@ -141,7 +141,7 @@
     "@storybook/react": "^5.0.11",
     "amphtml-validator": "^1.0.23",
     "babel-eslint": "^10.0.0",
-    "bbc-a11y": "^2.3.0",
+    "bbc-a11y": "latest",
```

Comment:
I don't think we should take beyond this major version, have discussed with @hindsc52

Reply:
Discussing this with @benjaminhobbs just now, he is right. We shouldn't be blindly installing the latest version of this package as we could very easily be introducing braking changes.

Reply:
fair enough :+1: